### ISSUE TITLE
calcite-modal: Poor color contrast ratio for "back" slot in dark theme

### ISSUE THREAD

**Issue by gpbmike** [Esri/calcite-components#4146]
### Actual Behavior

The text in `<calcite-button slot="back">` in `<calcite-modal>` is too dark when using the dark theme. (Contrast ratio 1.48:1)

![image](https://user-images.githubusercontent.com/8754/154724550-5e7025b7-806b-4d4a-9fed-260071d4e5f1.png)

### Expected Behavior

[Contrast ratio should be 4.5:1](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)

### Reproduction Sample

https://developers.arcgis.com/calcite-design-system/components/modal/

### Reproduction Steps

Turn on dark mode.

### Reproduction Version

beta.76

**Comment by macandcheese** [comment]
Thanks for reporting this @gpbmike - this was an instance of the documentation using an outdated value for the button color. There is a PR on the documentation site repo : https://github.com/ArcGIS/afd-calcite-documentation/pull/298

**Comment by gpbmike** [comment]
This looks good to me now. Thanks all.

![image](https://user-images.githubusercontent.com/8754/158863648-e81f489c-66df-4168-b430-828f7ef502d2.png)
